prepare/metrics/llm_as_judge_response_assessment.py

```diff
@@ -0,0 +1,21 @@
+from unitxt import add_to_catalog
+from unitxt.inference import PipelineBasedInferenceEngine
+from unitxt.llm_as_judge import LLMAsJudge
+
+inference_model = PipelineBasedInferenceEngine(
+    model_name="google/flan-t5-large", max_new_tokens=32
+)
+recipe = (
+    "card=cards.llm_as_judge.model_response_assessment.mt_bench,"
+    "template=templates.llm_as_judge.model_response_assessment.mt_bench,"
+    "demos_pool_size=0,"
+    "num_demos=0"
+)
+
+metric = LLMAsJudge(inference_model=inference_model, recipe=recipe)
+
+add_to_catalog(
+    metric,
+    "metrics.rag.llm_as_judge.model_response_assessment.mt_bench_flan_t5",
```

Comment:
How about this?

metrics.rag.model_response_assessment.llm_as_judge_by_flan_t5_using_mt_bench_template

I think the prefix should be the task and not the implementation.  Best if it's most readable.

Reply:
I would also add the inference engine (genai, watsonx, etc)
I will actually add a function that give you the name, so it kept under a certain convention

Reply:
Yes. How about this?

metrics.rag.model_response_assessment.llm_as_judge_by_flan_t5_large_on_bam_using_mt_bench_template

and inference_model.get(name) could return "flan_t5_large_on_bam".